physionet-django/user/models.py

```diff
@@ -827,6 +827,13 @@ def revoke(self):
         logger.info('Credentialing for user {0} has been removed.'.format(
             self.user.email))
 
+    def remove_contact_reference(self):
+        """
+        Mark a reference as not contacted making the reference for this instance
```

Comment:
Could this be reworded slightly to:

``` 
Remove the date that indicates when the reference was contacted. Note that this may also affect the "known" status of the reference.
```

Reply:
Fixed in next commit :)